bin/crew

```diff
@@ -43,6 +43,7 @@ DOC = <<~DOCOPT
     crew deps [options] [-t|--tree] [-b|--include-build-deps] [--exclude-buildessential] <name> ...
     crew download [options] [-s|--source] <name> ...
     crew files [options] <name> ...
+    crew getrealdeps <name> ...
```

Comment:
Would it make sense to integrate getrealdeps into the deps command?  I'm thinking having 2 commands might be a bit confusing.  At least for me. :blush:

Reply:
Deps is much quicker, because it doesn't need to extract and decompress everything... But that's also the downside...

Reply:
I was thinking we could add a switch.  Something like `-d` or `--deep` for a deep dive into dependencies.  It would do the same thing without the need to have a separate seemingly related command.

Reply:
I have added a `crew deps --deep` option.

Example: `crew deps --deep mpv`

Reply:
Do we need `crew getrealdeps` if we have `crew deps --deep`?

Reply:
Removed.